recipe/build_pytorch.sh

```diff
@@ -95,6 +100,18 @@ if [[ "$OSTYPE" == "darwin"* ]]; then
 fi
 
 if [[ ${cuda_compiler_version} != "None" ]]; then
+    # TODO: figure out what to patch to make this work without a CFLAGS hack
+    #   FAILED: caffe2/CMakeFiles/torch_cpu.dir/__/torch/csrc/jit/codegen/onednn/graph_fuser.cpp.o
+    #   /home/conda/feedstock_root/build_artifacts/pytorch-recipe_1690125468847/_build_env/bin/x86_64-conda-linux-gnu-c++ ...
+    #   /home/conda/feedstock_root/build_artifacts/pytorch-recipe_1690125468847/work/torch/csrc/jit/codegen/onednn/graph_fuser.cpp
+    #   In file included from /home/conda/feedstock_root/build_artifacts/pytorch-recipe_1690125468847/work/torch/csrc/jit/codegen/onednn/graph_fuser.h:3,
+    #                    from /home/conda/feedstock_root/build_artifacts/pytorch-recipe_1690125468847/work/torch/csrc/jit/codegen/onednn/graph_fuser.cpp:1:
+    #   /home/conda/feedstock_root/build_artifacts/pytorch-recipe_1690125468847/work/torch/csrc/jit/codegen/onednn/graph_helper.h:3:10: fatal error: oneapi/dnnl/dnnl_graph.hpp: No such file or directory
+    #       3 | #include <oneapi/dnnl/dnnl_graph.hpp>
+    #         |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    export CFLAGS="${CLFAGS} -I${SRC_DIR}/third_party/ideep/mkl-dnn/include/"
+    export CXXFLAGS="${CXXLFAGS} -I${SRC_DIR}/third_party/ideep/mkl-dnn/include/"
```

Comment:
ok i'm going to try to see if 1 fixes things. the issue seems to be on the linking side. so i'm hoping that it does!

Reply:
we also need to finish the protobuf migration too.

so maybe some help there? it seems like we would need to point to the build protoc instead of the host protoc for cross compilation processes not sure if there is an upstream issue

Reply:
I do not think these includes are needed.
The pytorch build scripts do not do that.
I believe the original issues we were seeing were due to the mkl version being installed by conda.
Pytorch pins the version to 2021.4.0, see here https://github.com/pytorch/pytorch/blob/0ab74044c2775970d3bc3668454a3152ae18ea82/.ci/docker/common/install_conda.sh#L54